Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/ValueStringFilter.h

```diff
@@ -18,19 +18,19 @@ namespace AZ::DocumentPropertyEditor
         ValueStringFilter();
         void SetFilterString(AZStd::string filterString);
 
+    protected:
+        // pure virtual overrides
+        virtual MatchInfoNode* NewMatchInfoNode() const override;
+        virtual void CacheDomInfoForNode(const Dom::Value& domValue, MatchInfoNode* matchNode) const override;
+        virtual bool MatchesFilter(MatchInfoNode* matchNode) const override;
```

Comment:
The `virtual` keyword should be removed.

Reply:
done